tests/test_api.py

```diff
@@ -446,45 +446,72 @@ def test_preprocess_error(self):
         processor = api.GradeCSVProcessor(course_id=self.course_id)
         row = {
             'block_id': self.usage_key,
+            'new_override-123402db': '1',
             'new_override-85bb02db': '-1',
             'user_id': self.learner.id,
             'csum': '07ec',
             'Previous Points': '',
         }
-        with self.assertRaisesMessage(ValidationError, 'Grade must be positive'):
+        with self.assertRaisesMessage(ValueError, 'Grade must be positive'):
             processor.preprocess_row(row)
         row['new_override-85bb02db'] = 'not a number'
         with self.assertRaisesMessage(ValidationError, 'Grade must be a number'):
             processor.preprocess_row(row)
 
-    def test_repeat_user(self):
+    def test_multiple_subsection_override(self):
         processor = api.GradeCSVProcessor(course_id=self.course_id)
         row = {
             'block_id': self.usage_key,
+            'new_override-12f402db': '3',
+            'new_override-123402db': '2',
             'new_override-85bb02db': '1',
             'user_id': self.learner.id,
             'Previous Points': '',
         }
         operation = processor.preprocess_row(row)
-        assert len(operation) == 4
+        assert len(operation) == 3
```

Comment:
This is how the existing tests do it, but it's not how we should be doing it.

from [PEP-8](https://www.python.org/dev/peps/pep-0008/#id51):
```
For sequences, (strings, lists, tuples), use the fact that empty sequences are false:

# Correct:
   if not seq:
   if seq:
# Wrong:
   if len(seq):
   if not len(seq):
```

So let's make this and line 485 `assert operation` and make 496 `assert not operation`